react-jsx/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-jsx",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-transform-react-jsx": "^7.14.5",
+    "babel-loader": "^8.2.2",
+    "webpack": "^5.43.0",
+    "webpack-cli": "^4.7.2"
+  },
+  "scripts": {
```

Comment:
Put any new scripts in the existing script object above, this will overwrite any previous scripts. There should only be one script object